core/workspace_svg.js

```diff
@@ -1348,6 +1348,15 @@ Blockly.WorkspaceSvg.prototype.zoomCenter = function(type) {
   var metrics = this.getMetrics();
   var x = metrics.viewWidth / 2;
   var y = metrics.viewHeight / 2;
+  if (this.options.toolboxPosition == 2) {
```

Comment:
I think @rachel-fenichel means to use a constant rather than a magic number to test this. The constants for the toolbox position are defined here: https://github.com/LLK/scratch-blocks/blob/d7745b12fdb5ee9a4761b7e99fd201673e0e7a8e/core/constants.js#L192-L214